SPECS/azurelinux-release/90-default.preset

```diff
@@ -0,0 +1,184 @@
+
+disable systemd-timesyncd.service
+
+enable systemd-networkd.service
+enable systemd-networkd-wait-online.service
+
+disable systemd-boot-update.service
+
+enable systemd-resolved.service
+
+disable systemd-oomd.service
+
+enable dbus.socket
+enable dbus.service
+enable sshd.service
+
+enable crond.*
+
+enable chronyd.service
+
+enable auditd.service
+
+enable restorecond.service
+
+enable avahi-daemon.*
+
+enable rsyslog.*
+enable syslog-ng.*
+enable sysklogd.*
+
+enable firewalld.service
+
+enable virtqemud.socket
+enable virtqemud-ro.socket
+enable virtqemud-admin.socket
+enable virtvboxd.socket
+enable virtvboxd-ro.socket
+enable virtvboxd-admin.socket
+
+enable virtproxyd.socket
+enable virtproxyd-ro.socket
+enable virtproxyd-admin.socket
+
+enable virtinterfaced.socket
+enable virtinterfaced-ro.socket
+enable virtinterfaced-admin.socket
+enable virtnetworkd.socket
+enable virtnetworkd-ro.socket
+enable virtnetworkd-admin.socket
+enable virtnodedevd.socket
+enable virtnodedevd-ro.socket
+enable virtnodedevd-admin.socket
+enable virtnwfilterd.socket
+enable virtnwfilterd-ro.socket
+enable virtnwfilterd-admin.socket
+enable virtsecretd.socket
+enable virtsecretd-ro.socket
+enable virtsecretd-admin.socket
+enable virtstoraged.socket
+enable virtstoraged-ro.socket
+enable virtstoraged-admin.socket
+
+enable virtlogd.socket
+enable virtlogd-admin.socket
+enable virtlockd.socket
+enable virtlockd-admin.socket
+
+enable xinetd.service
+
+enable multipathd.service
+
+enable libstoragemgmt.service
+
+enable iscsid.socket
+enable iscsiuio.socket
+
+enable iscsi-starter.service
+
+enable iscsi-onboot.service
+
+enable lvm2-monitor.*
+enable lvm2-lvmetad.*
+enable dm-event.*
+enable dmraid-activation.service
+
+enable mdmonitor.service
+enable mdmonitor-takeover.service
+
+enable raid-check.timer
+
+enable spice-vdagentd.service
+
+enable qemu-guest-agent.service
```

Comment:
why is this enabled by default? 
AFAIK the common usage is not qemu VMs. This should only be enabled in the `qemu-guest` references image

Reply:
These are simply the presets, these don't mean the package will actually be installed.

For `qemu-guest-agent` in particular, you can see it's `WantedBy=dev-virtio\x2dports-org.qemu.guest_agent.0.device` so, *when enabled*, it will only register itself to actually be *started* when that virtual device is detected in the system. Meaning that even if the package is installed onto a system, and the service is enabled, it won't actually get started unless it's actually a qemu guest instance.

I have no particularly strong feeling about any of the presets (taken mostly from fedora's presets), and I also have no problem with specific image builds adding (or overriding) any presets. However this one in particular seems to make sense to leave enabled here, unless I'm missing something.